src/Common/src/TypeSystem/IL/Stubs/InterlockedIntrinsic.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using Internal.TypeSystem;
+
+using Debug = System.Diagnostics.Debug;
+
+namespace Internal.IL.Stubs
+{
+    /// <summary>
+    /// Provides method bodies for generic Interlocked intrinsics. These intrinsics work around the lack of byref locals 
+    /// return values in C#. The intrinsic method is forwards the call to non-generic version.
```

Comment:
'is' should be removed.  And since I'm being really nit-picky, you should also add 'the' in front of "non-generic".